README.md

```diff
@@ -218,8 +219,8 @@ and `{}` for a mutually exclusive keyword.
 | Commit Read-Write Transaction | `COMMIT;` | |
 | Rollback Read-Write Transaction | `ROLLBACK;` | |
 | Start Read-Only Transaction | `BEGIN RO [{<seconds>\|<RFC3339-formatted time>}] [PRIORITY {HIGH\|MEDIUM\|LOW}] [TAG <tag>];` | `<seconds>` and `<RFC3339-formatted time>` is used for stale read. See [Request Priority](#request-priority) for details on the priority. The tag you set is used as request tag. See also [Transaction Tags and Request Tags](#transaction-tags-and-request-tags).|
-| End Read-Only Transaction | `CLOSE;` | |
-| Exit CLI | `EXIT;` | |
+| End Read-Only Transaction | `CLOSE;`                                                                                       | |
```

Comment:
Is this changed by some formatter?

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/173/commits/0d7393de10bf1a597b0276803a03ec9da1761dca